I wish_If only/vova_timonin.md

```diff
@@ -0,0 +1,29 @@
+1. Жаль что вы не пришли пораньше, чтобы оценить всю ущербность архитектуры предложенной студентом первокурсником.
+I wish you came earlier to evaluate imperfect architecture the first-year student had proposed.
+
+2. Обидно, что мы не должались его прихода, иначе бы мы были в курсе на каком этапе сейчас находится проект и что нам делать с этим дальше.
+I wish we waited his arrival, otherwise we have been aware what stage of the progect is now and what we should do with it next.
```

Comment:
I wish we managed to wait till his arrival
otherwise we would be aware of the stage the project is currently on